app/run.py

```diff
@@ -49,4 +49,4 @@ def on_get(self, req, resp):
     app.add_route('/app/elasticsearch', ElasticsearchResource())
 app.add_route('/app', BaseResource())
 app.add_route('/app/busy', BusyResource())
-app.add_route('/ping', PingResource())
```

Comment:
the app (uwsgi) handles `/ping` under `/app/ping` so that nginx can also handle a `/ping`